src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -5946,6 +5966,19 @@ public ViewLayoutDirectionType Type
             }
         }
 
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
please add "[EditorBrowsable(EditorBrowsableState.Never)]" before ACR, otherwise it will break the API-Compatibility